Support for enabling/disabling External Gateway Client certificate inclusion in HTTP headers (#2640)
kyma-project/api-gateway: docs/release-notes/3.7.0.md

## New Features
- DNSEntry custom resources now support the DualStack IP stack. The operator detects the IP stack of the `istio-ingressgateway` Service by inspecting the **spec.ipFamilies** field and sets the appropriate annotation on the DNSEntry resource. See [issue #2587](https://github.com/kyma-project/api-gateway/issues/2587).

> - Added a new field includeExtGatewayClientCert to the ExternalGateway spec, which allows including ExternalGateway client certificate in HTTP headers.

Reviewer: ```suggestion
- The ExternalGateway **spec** now includes the **includeExtGatewayClientCert** field, which allows you to include the ExternalGateway client certificate in HTTP headers.
```
